hw3/my_gem/app/application.rb

```diff
@@ -0,0 +1,26 @@
+require 'my_gem/version'
+##
+# Controller (inherit from our main Controller class)
+#
+class TestsController < Controller
```

Comment:
if you need this test applicartion for testing you should place it to `spec/fixtures/test_app.rb`